connector/src/main/java/org/jboss/as/connector/subsystems/jca/CachedConnectionManagerRemove.java

```diff
@@ -0,0 +1,28 @@
+package org.jboss.as.connector.subsystems.jca;
+
+import org.jboss.as.controller.AbstractRemoveStepHandler;
+import org.jboss.as.controller.OperationContext;
+import org.jboss.as.controller.OperationFailedException;
+import org.jboss.dmr.ModelNode;
+import org.jboss.dmr.ModelType;
+
+/**
+ * @author <a href="ingo@redhat.com">Ingo Weiss</a>
+ */
+class CachedConnectionManagerRemove extends AbstractRemoveStepHandler {
```

Comment:
This probably shouldn't be an extension of any standard remove step handler, as it's very custom. See the last commit on https://github.com/bstansberry/wildfly/commits/pullRequest9206 for how the handler should look. Sorry; I should have provided that right off. I wasn't thinking about how custom this is.